bin/bout-boutdata-restart-create

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env python3
+# PYTHON_ARGCOMPLETE_OK
+
+import argparse
+try:
+    import argcomplete
+except ImportError:
+    argcomplete=None
+
+parser = argparse.ArgumentParser("Create restart files from data (dmp) files.")
+parser.add_argument("path", type=str, default=...
+    ,help="Path to original restart files (default: \"data\")")
```

Comment:
Use the `default` argument to specify the default, rather than the help string. Also, don't escape quotes in nested quotes, just use the "other" one (single or double quotes)

Reply:
But then I need to mix the quotes. And I need to read the string, checking which quotes are in there, and depending on that still need to escape ...
Is it worth all the trouble?

Reply:
I am now checking what quotes are in the string, so here single quotes are used.